layouts/New/Blog.tsx

```diff
@@ -38,13 +39,9 @@ const BlogLayout: FC = async () => {
   return (
     <>
       <WithNavBar />
-
```

Comment:
Why removing these extra spaces? They are intentional to make it easier to read the file / separation of stuff

Reply:
> They are intentional to make it easier to read the file / separation of stuff

😁 it's funny but I'd rather not have any extra space because at least you have everything together. And it makes it easier to see the JSX structure (IMHO).

So just keep extra spaces or not ?

Reply:
As usual, let's avoid unrelated changes even more ones that are down to personal opinions. We can always open a discussion about what would be better to be adopted.

Reply:
OK !